src/components/_common/Input/Input.tsx

```diff
@@ -0,0 +1,20 @@
+import S from "./style";
+import { forwardRef } from "react";
+interface Props extends React.HTMLProps<HTMLInputElement> {
+  isError: boolean;
+}
+
+const Input = forwardRef<HTMLInputElement, Props>(
+  ({ isError, ...restProps }, ref) => {
+    return (
+      <S.InputBox
+        ref={ref}
+        $isError={isError}
+        maxLength={2}
+        {...restProps}
```

Comment:
default값이 있다면 명시적으로 주는 게 좋겠습니다.
```suggestion
  ({ isError, maxLength = 2, ...restProps }, ref) => {
    return (
      <S.InputBox
        ref={ref}
        $isError={isError}
        maxLength={maxLength}
        {...restProps}
```

Reply:
좋은 방식인 것 같습니다!!